src/app/api.py

```diff
@@ -159,66 +126,21 @@ def ping():
     return {"status": "ok"}
 
 
-@app.get(
-    "/get_progress",
-    description="Returns progress of the metadata extraction. From 0 to 1 (=100%).",
-    response_model=ProgressOutput,
-)
-def get_progress(progress_input: ProgressInput):
-    if progress_input.url != "" and shared_status[1] != "" and shared_status[1] in progress_input.url:
-        progress = round(shared_status[0] / NUMBER_OF_EXTRACTORS, 2)
-    else:
-        progress = -1
-    return {
-        "progress": progress,
-    }
-
-
 # Developer endpoints
 if not is_production_environment():
 
-    @app.get(
-        "/records",
-        response_model=RecordsOutput,
-        description="Get all urls and their processed metadata.",
-    )
-    def show_records():
-        records = load_records()
-        out = []
-        for record in records:
-            out.append(
-                RecordSchema(
-                    id=record.id,
-                    timestamp=record.timestamp,
-                    start_time=record.start_time,
-                    action=record.action,
-                    allow_list=record.allow_list,
-                    meta=record.meta,
-                    url=record.url,
-                    html=record.html,
-                    headers=record.headers,
-                    har=record.har,
-                    debug=record.debug,
-                    exception=record.exception,
-                    time_until_complete=record.time_until_complete,
-                )
-            )
-        return {"records": out}
-
     @app.get(
         "/cache",
         response_model=CacheOutput,
         description="Developer endpoint to receive cache content.",
     )
     def get_cache():
-        return {"cache": load_cache()}
+        return CacheOutput()  # fixme
```

Comment:
Yes, I actually would like to keep this somehow. That would allow a consumer of the service to quickly get all results that have been processes in the (recent) past without having to wait. As the results may get pretty big here, we might want some pagination or similar.

Reply:
These are multiple points, which include additional features, e.g., pagination. IMHO that goes beyond a simple fixme. Nevertheless, I guess you know what you will do here, so this is fine with me :)